### DISCUSSION TITLE
Customize Menu Image

### DISCUSSION THREAD

**Discussion by sup3rm4nny** [Godiesc/firefox-one#44] [General]
Hello, in the preview page for "Image in Menus" you mention that you can either get a solid theme menu image or one that is adapted to the image of the theme. When I choose a solid theme the image comes out correctly but when I select the other option for adapted to the image of the theme the menu image comes out very blurry. How do I change it so it looks like the example that you provided with a clear image? I attached a photo.

Firefox-one is up to date and I am using your theme called "Beautiful Red Blur". 

![Screenshot 2024-09-10 172749](https://github.com/user-attachments/assets/6a00398d-5d32-40d9-8397-d0970169ca1a)

**Comment by Godiesc** [comment]
hi, 
sorry for the late response, that is the right way the theme display the image, sadly most themes usually has images for example of 400px of height, and to display that image in menus bigger than that the image will repeat and depending of the image it is going to repeat itself horribly, for example: 

![imagen](https://github.com/user-attachments/assets/160298c9-3f75-4e3f-9386-b68b5fbc177f)

you can see the image repeat and don't seen right, for that I expand the image and that cause the blurry effect. few themes has a bigger image for example this that is the one used in the preview: [galaxy_cool_thing](https://addons.mozilla.org/es/firefox/addon/galaxy_cool_thing/)
